knowledge-graph/src/main/scala/ch/datascience/knowledgegraph/datasets/rest/BaseDetailsFinder.scala

```diff
@@ -61,11 +61,23 @@ private class BaseDetailsFinder(
         |    ?datasetId schema:identifier "$identifier";
         |               schema:identifier ?identifier;
         |               rdf:type <http://schema.org/Dataset>;
+        |               schema:isPartOf ?projectId; 
         |               schema:url ?url;
         |               schema:name ?name;
         |               schema:alternateName ?alternateName;
+        |               prov:atLocation ?location ;
         |               renku:topmostSameAs ?topmostSameAs;
         |               renku:topmostDerivedFrom/schema:identifier ?initialVersion.
+        |               
+        |    BIND(CONCAT(?location, "/metadata.yml") AS ?metaDataLocation) .
+        |    FILTER NOT EXISTS {
+        |      # Removing dataset that have an activity that invalidates them
+        |      ?deprecationEntity rdf:type <http://www.w3.org/ns/prov#Entity>;
+        |                         prov:atLocation ?metaDataLocation ;
+        |                         prov:wasInvalidatedBy ?invalidationActivity ;
```

Comment:
Do we use this invalidationActivity anywhere or is there an equivalent of _ so we can ignore it?

Reply:
Yes this means that I want the prov:wasInvalidatedBy to be pointing to something